spyderlib/widgets/fileswitcher.py

```diff
@@ -443,7 +443,11 @@ def goto_line(self, line_number):
     # --- Helper methods: Outline explorer
     def get_symbol_list(self):
         """Get the object explorer data."""
-        return self.get_editor().highlighter.get_outlineexplorer_data()
+        try:
+            oedata = self.get_editor().highlighter.get_outlineexplorer_data()
```

Comment:
Seeing this long call makes me wonder, why we dont have a simpler method... like

```python
self.outlineexplorer_data()
```
not a fan of `get` in methods, this is part of my argument against using PEP8 for naming stuff... instead of Qt conventions. Qt conventions are quit nice and clear.